dist/index.d.ts

```diff
@@ -458,7 +458,10 @@ interface GPUBindGroupEntry {
 
 interface GPUBindGroupLayoutDescriptor
   extends GPUObjectDescriptorBase {
-  entries: Iterable<GPUBindGroupLayoutEntry>;
+  /**
+   * A list of entries describing the shader resource bindings for a bind group.
+   */
+  entries: Array<GPUBindGroupLayoutEntry>;
```

Comment:
This should stay as `Iterable`.
```suggestion
  entries: Iterable<GPUBindGroupLayoutEntry>;
```